src/ts/plugins/grapesjs-theme-mode.ts

```diff
@@ -0,0 +1,105 @@
+//TEST 10
+import grapesjs, { Editor, PluginOptions, CssRule } from 'grapesjs'
+
+export default (editor: Editor, opts: PluginOptions = {}) => {
+  //console.log('Plugin grapesjs-theme-mode chargé !');
+
+  //define the current mode to both
+  (editor as any).ThemeMode = { current: 'both' }
```

Comment:
You don't need to add a property to editor in this case

The best would be to not keep the state anywhere, but if you really need to, keep the state in a global variable

You can also fire events on `editor` if you need other parts to know about this state